src/MongoDB.Driver.Core/Core/Clusters/MultiServerCluster.cs

```diff
@@ -323,7 +323,8 @@ private void ProcessServerDescriptionChanged(ServerDescriptionChangedEventArgs a
                     }
                 }
 
-                UpdateClusterDescription(newClusterDescription);
+                var shouldClusterDescriptionChangedEventBePublished = !args.OldServerDescription.SdamEquals(args.NewServerDescription);
```

Comment:
Is there anywhere in the spec that specifies this logic should happen? I'm guessing it's the table here? https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#events

Also, is there a spec test for this logic? I would imagine that the `Monitoring a standalone connection - suppress update events for equal server descriptions"` results in a SingleServerCluster being created?

Reply:
>Is there anywhere in the spec that specifies this logic should happen?

do you mean the spec requirements about where we need to trigger such events?

>Also, is there a spec test for this logic? 

no, but it sounds like a good idea to add it, I will work on it